scripts/v.report/v.report.py

```diff
@@ -217,7 +217,7 @@ def main():
         sys.stdout.write(fs.join(colnames + extracolnames) + "\n")
 
     # make and print the table:
-    numcols = len(colnames) + len(extracolnames)
+    numcols = len(colnames) + len(extracolnames)  # noqa: F841
```

Comment:
Same check of the surrounding source code as for table above would be worth it, but here I would expect it is simply not needed unless something like `len(colnames)` is used somewhere in the code.

Reply:
Even in September 2008, 16 years ago (first commit), the variable numcols seems unused https://github.com/OSGeo/grass/blob/14ec4aeffa93547fc3cf0af59312f38128e605bc/scripts/v.report/v.report.py

So why couldn't we just finally remove it.

What bothers me is the comment above that mentions making and printing a table, but I don't see how it was ever done within that "paragraph". Way below with the list comprehensions it might be possible that it makes it.

Reply:
See here where `numcols` came from:

https://github.com/OSGeo/grass-legacy/blob/2734c86fd5cb976b4a94b04a2cdc75b4613f6a77/scripts/v.report/v.report#L168

(and lines below)

Reply:
So it isn't used in there anymore, safe to go away.